forestci/tests/test_forestci.py

```diff
@@ -2,6 +2,15 @@
 import numpy.testing as npt
 from sklearn.ensemble import RandomForestRegressor
 import forestci as fci
+import forestci.cyfci
+
+
+def test_compare_cycore_computation():
+    a = np.arange(1,7,dtype=np.float64).reshape(2,3)
+    b = np.arange(1,13,dtype=np.float64).reshape(4,3)
+    c = fci._cycore_computation(a, b)
+    actual = fci._core_computation(np.zeros((2,10)), np.zeros((4,10)), a, b, 3)
+    npt.assert_equal(actual, c)
```

Comment:
```suggestion
    npt.assert_almost_equal(actual, c)
```

Reply:
@owlas : if you approve this suggested change, I think that this should be ready to go.